src/gpu/nvidia/cudnn_reorder_impl.hpp

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
-* Copyright 2020-2023 Intel Corporation
-* Copyright 2020 Codeplay Software Limited
+* Copyright 2024 Intel Corporation
+* Copyright 2024 Codeplay Software Limited
```

Comment:
If no change to this file, please drop the copyright banner change.

Reply:
Copyright has been fixed.